rokwire.yaml

```diff
@@ -4,7 +4,7 @@ info:
   description: REST API design and documentation for Rokwire platform
   version: 0.0.1
 servers:
-- url: https://virtserver.swaggerhub.com/uiuc-scci/rokwire/0.0.1
+- url: https://virtserver.swaggerhub.com/uiuc-rokwire/rokwire/0.0.1
```

Comment:
This line was generated automatically by swagger tool (Swaggerhub in this case) during design. Not sure if we should keep it here because if I point to the server, it does not really exist. What do you think?